crates/macros/Cargo.toml

```diff
@@ -4,7 +4,7 @@ description = "Derive macros for ext-php-rs."
 repository = "https://github.com/davidcole1340/ext-php-rs"
 homepage = "https://github.com/davidcole1340/ext-php-rs"
 license = "MIT OR Apache-2.0"
-version = "0.10.2"
```

Comment:
Please don't manually update the version numbers. This is done by our release process.